stream_alert/rule_processor/rules_engine.py

```diff
@@ -253,6 +253,9 @@ def process(cls, input_payload):
                 # rule analysis
                 rule_result = cls.process_rule(record, rule)
                 if rule_result:
+                    LOGGER.info('Rule triggered [%s] on log type [%s] from entity \'%s\' '
```

Comment:
What does "triggered" mean? The rule was evaluated, or the rule evaluated to True?

Reply:
@mime-frame  In this case yeah - do you have a suggestion for better verbiage?

Reply:
Alert fired?

Reply:
Rule fired?

Reply:
> In this case yeah

lol, which one :)

Reply:
Sorry missed that bit - what is `function decl`?

Reply:
function decl = function declaration = all of the requirements list in the function declaration of the rule = `req_subkeys`, `logs`, ...

Reply:
gotcha - in that case, b - since b cannot be true without a. This log message is supposed to indicate that an alert has been fired and should be making its way to the alert processor via the rule > alert lambda invocation. There hasn't been a logger statement that does this until now.

Reply:
Awesome, thanks for clarifying!  Perhaps then `Rule [%s] triggered an alert on log type [%s]...`

Reply:
okay - that works for me!